modules/pel/peltool/peltool.py

```diff
@@ -211,29 +211,35 @@ def severityBasedExclusion(uh: UserHeader, config: Config) -> bool:
             return False
     return True
 
-def excludePEL(uh: UserHeader, config: Config) -> bool:
+def includePEL(uh: UserHeader, config: Config) -> bool:
     """
     Evaluates whether a PEL meets criteria based on specified conditions.
-    Returns True if the entry should be Excluded, False otherwise.
+    Returns True if the entry should be Included, False otherwise.
     """
-    if config.critSysTerm and uh.eventSeverity != SeverityValues.critSysTermSeverity.value:
+    if config.critSysTerm and uh.eventSeverity == SeverityValues.critSysTermSeverity.value:
         return True
-    
-    if config.non_serviceable_only and uh.isServiceable():
+
+    if config.non_serviceable_only and not uh.isServiceable():
         return True
     
-    if config.serviceable_only and not uh.isServiceable():
+    if config.serviceable_only and uh.isServiceable():
         return True
+
+    if config.severities and not severityBasedExclusion(uh, config):
+        return True
+
+    if config.hidden and uh.actionFlags & ActionFlagsValues.hiddenActionFlag.value:
+        return True
+
+    # Having verified each condition individually, reaching this point indicates that this PEL does not satisfy the criteria for any selected option. 
```

Comment:
I believe you are trying to determine whether the serviceable PELs need to be returned by checking the below conditions, correct?

If so, the logic could be something like this:

```
"""
Having verified each condition individually, reaching this point
indicates that the PEL does not satisfy the criteria for any
selected option. Therefore, check if the "--only" option is used:
If "Yes", the PEL cannot be included.
If "No", return serviceable logs as the default case (i.e., hidden or
non-serviceable PELs should not be included).
"""
if config.only:
   return False

if uh.actionFlags & ActionFlagsValues.hiddenActionFlag.value or
   not uh.isServiceable():
   return False

return True
```

FYI, code/comments should not exist 75 characters as per guide line so added with multi-line comments

Reply:
I think we need to cover all cases as mentioned [here](https://github.com/RameshIyyar/openpower-pel-parsers/blob/tool_transition_impacts/tool_transition_impacts.md#tool-usage)

Could please check if you are not covered?

Reply:
Working on few examples I tested with!